src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/ReadsFilter.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+
+/**
+ * A simple filter for reads. Pulled into its own class due to Java limitations.
```

Comment:
Remove comment "Pulled into its own class due to Java limitations" -- I think we'd want this as its own class regardless of Java limitations :)